crypto/ed25519/ed25519_test.go

```diff
@@ -53,3 +59,48 @@ func TestBatchSafe(t *testing.T) {
 	ok, _ := v.Verify()
 	require.True(t, ok)
 }
+
+func TestFromDer(t *testing.T) {
+	var testCases = []struct {
+		privkeyBase64Der        string
+		expectedPubkeyBase64Der string
+	}{
+		{
+			privkeyBase64Der:        "MC4CAQAwBQYDK2VwBCIEIB/3MZ9V0e8JidiOiDtN3Nk3sGnwohSgaAmIFuScDfOy",
+			expectedPubkeyBase64Der: "MCowBQYDK2VwAyEAcpYVXaxQmDGUnlpgTe71OKv4cUcbw8k+/IeW8cZF4W4=",
+		},
+	}
+
+	for _, tc := range testCases {
+		t.Run("", func(t *testing.T) {
+			privkeyDer, err := base64.StdEncoding.DecodeString(tc.privkeyBase64Der)
+			require.NoError(t, err)
+			expectedPubkeyDer, err := base64.StdEncoding.DecodeString(tc.expectedPubkeyBase64Der)
+			require.NoError(t, err)
+
+			expectedPubkeyStd, err := x509.ParsePKIXPublicKey(expectedPubkeyDer)
+			require.NoError(t, err)
+			expectedPubkey := []byte(expectedPubkeyStd.(stded25519.PublicKey))
+
+			privkey, err := ed25519.FromDER(privkeyDer)
+			assert.NoError(t, err)
+			assert.Len(t, privkey, ed25519.PrivateKeySize)
+			assert.Equal(t, expectedPubkey, privkey.PubKey().Bytes())
+		})
+	}
+}
+
+// TestEd25519StdlibCompat ensures that key format in "crypto/ed25519" is compatible with
+// the keys in "github.com/oasisprotocol/curve25519-voi" (which we use)
+func TestEd25519StdlibCompat(t *testing.T) {
```

Comment:
IMO, this is a redundant test because it doesn't test any of the internal functions